src/briefcase/commands/dev.py

```diff
@@ -18,6 +18,18 @@ class DevCommand(RunAppMixin, BaseCommand):
     output_format = None
     description = "Run a Briefcase project in the dev environment."
 
+    DEV_ENVIRONMENT = {
```

Comment:
```suggestion
    # On macOS, a document-based app reads the raw argc/argv that is used to start the
    # process. This means we can't use any command line arguments when invoking the
    # python executable, because those will leak into the running app as "documents"
    # that are to be opened on startup. These arguments *can* be passed in as command
    # line arguments, but are configured using environment variables.
    DEV_ENVIRONMENT = {
```